ui/src/hooks/helper/index.ts

```diff
@@ -125,35 +138,11 @@ export const handleCustomKey = (
   lunaId: string,
   origin: string
 ): boolean => {
-
-  const debouncedSwitchTab = useDebounceFn((lunaId: string, origin: string, key: string) => {
-    switch (key) {
-      case 'ArrowRight':
-        sendEventToLuna('KEYEVENT', 'alt+shift+right', lunaId, origin);
-        break;
-      case 'ArrowLeft':
-        sendEventToLuna('KEYEVENT', 'alt+shift+left', lunaId, origin);
-        break;
-    }
-  }, 500);
-
   if (e.altKey && e.shiftKey && (e.key === 'ArrowRight' || e.key === 'ArrowLeft')) {
-    switch (e.key) {
-      case 'ArrowRight':
-        if (lunaId && origin) {
-          debouncedSwitchTab(lunaId, origin, 'ArrowRight');
-        } else {
-          mittBus.emit('alt-shift-right');
-        }
-
-        break;
-      case 'ArrowLeft':
-        if (lunaId && origin) {
-          debouncedSwitchTab(lunaId, origin, 'ArrowLeft');
-        } else {
-          mittBus.emit('alt-shift-left');
-        }
-        break;
+    if (lunaId && origin) {
+      debouncedSwitchTab(lunaId, origin, e.key);
+    } else {
+      mittBus.emit(e.key === 'ArrowRight' ? 'alt-shift-right' : 'alt-shift-left');
     }
     return false;
   }
```

Comment:
The code is well-written with minimal changes needed to keep it efficient across different browsers and devices. There are no major issues noted thus far. However, for further improvements you might want to look into reducing DOM mutations or implementing lazy loading where necessary.